src/EditorFeatures/Core/Implementation/GoToDefinition/GoToDefinitionHelpers.cs

```diff
@@ -20,9 +20,9 @@ internal static class GoToDefinitionHelpers
             ISymbol symbol,
             Project project,
             IEnumerable<Lazy<INavigableItemsPresenter>> presenters,
-            ITypeSymbol containingTypeSymbol,
-            bool throwOnHiddenDefinition,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken,
+            bool thirdPartyNavigationAllowed = true,
+            bool throwOnHiddenDefinition = false)
```

Comment:
Isn't there an analyzer that requires the CancellationToken to be the last parameter, even when there are default parameters?